business_requirement_from_support/models/project_issue.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# © 2017 Praxya (https://www.praxya.com).
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, fields, models, _
+
+
+class ProjectIssue(models.Model):
+    _inherit = "project.issue"
+
+    business_requirement_id = fields.Many2one(
```

Comment:
The answer to the first one is yes, I just forgot, so I guess the button should be hidden if the issue has a related BR already, do you agree?